### ISSUE TITLE
Exact code

### ISSUE THREAD

**Issue by WombleWoo7547** [CodingGarden/sproutkit#12]
Could you have a branch where the exact code your overlay uses is there bc it doesnt work for me

**Comment by w3cj** [comment]
All of the code is on the main branch. It is what I'm using locally. What errors are you getting?

**Comment by WombleWoo7547** [comment]
No errors, just an invisible electron app that doesn't even have its own window??

**Comment by w3cj** [comment]
Yeah that is expected. The window starts as invisible / transparent and only shows chat messages from the last 24 hours. If you hover your mouse over it, you should see the toolbar.
